docs/team/nealetham.md

```diff
@@ -62,7 +62,7 @@ I reviewed the following pull requests:
 3. [Edit README #28](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/28)
 4. [Update index.md #29](https://github.com/AY2223S1-CS2103T-W12-3/tp/pull/29)
 
-I also gave guidance to my team on the following matters:
+I also gave guidance to my team on the following matters: test
```

Comment:
I think you might have been using the test branch while implementing this.

It's a small issue, you can revert those commits. If you want to learn more about how you can revert the commits click [here](https://www.theserverside.com/tutorial/How-to-git-revert-a-commit-A-simple-undo-changes-example#:~:text=Steps%20to%20revert%20a%20Git%20commit&text=Locate%20the%20ID%20of%20the,why%20the%20revert%20was%20needed.).

![image](https://user-images.githubusercontent.com/95712150/196428696-f77d1e95-3b43-4fbf-8cec-6c09d9cff483.png)